### DISCUSSION TITLE
Resizable Chat-Overview Sidebar (Left) for Better Visibility of Long Titles

### DISCUSSION THREAD

**Discussion by phischde** [open-webui/open-webui#7909] [Ideas]
### Discussed in https://github.com/open-webui/open-webui/discussions/7765

<div type='discussions-op-text'>

<sup>Originally posted by **phischde** December 11, 2024</sup>
I have noticed that while it is possible to resize the Chat Controls sidebar on the right, the Chat-Overview Sidebar on the left remains fixed in size. This can cause issues for users who need to view the full title of chats, especially in languages like German, where compound nouns or longer titles are common.

I propose adding the ability to resize the left Chat Sidebar to accommodate varying title lengths. This would enhance the usability of the interface and improve accessibility, particularly for users working in languages with naturally longer text.</div>

**Comment by EricLie-c** [comment]
Yes I fully agree. I tried out to find a method to make the side-bar wider but just find this issue lol.

**Comment by jonny255** [comment]
I concur. Looking for such a solution/feature as well.

**Comment by Classic298** [comment]
was implemented some time ago
